pace-util/pace/util/global_config.py

```diff
@@ -11,8 +11,6 @@ def getenv_bool(name: str, default: str) -> bool:
 def set_backend(new_backend: str):
     global _BACKEND
     _BACKEND = new_backend
-    for function in (is_gpu_backend, is_gtc_backend):
```

Comment:
Can we delete this file entirely now (possibly in a later PR)?

Reply:
Yes, in a follow up this afternoon!